ts/components/session/settings/SessionSettingListItem.tsx

```diff
@@ -91,17 +91,18 @@ export class SessionSettingListItem extends React.Component<Props, State> {
           {type === SessionSettingType.Slider && (
             <div className="slider-wrapper">
               <Slider
-                dots={true}
-                step={6}
-                min={12}
-                max={96}
-                defaultValue={currentSliderValue}
+                dots={content.dotsEnable}
+                step={content.step}
+                min={content.min}
+                max={content.max}
+                defaultValue={content.defaultValue}
```

Comment:
```suggestion
                defaultValue={currentSliderValue}
```
Just setting this back to what it was